doc/governance.rst

```diff
@@ -71,43 +71,39 @@ those aspects of the project.
 
 To this end, they have the required permissions on github to label and close
 issues. :ref:`Their work <bug_triaging>` is crucial to improve the
-communication in the project and limit the crowding of the issue tracker.
+communication in the project and limit the crowding of the issue tracker:
```

Comment:
This is not where the bullet list starts :)
The first item should be the contributor experience team just above

Reply:
Indded. Should be fixed